hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactNullablePrimitiveInteroperabilityTest.java

```diff
@@ -135,6 +137,64 @@ public void testWritePrimitiveReadNullable() {
         assertReadAsNullable(serializedRecord);
     }
 
+    @Test
+    public void testWritePrimitiveReadNullableCustomSerializer() {
+        FixedFieldsDTO fixedFieldsDTO = createFixedFieldsDTO();
```

Comment:
minor: I would replace `FixedFields` with `FixedSizeFields` to make it more clear.